src/streamsync/core.py

```diff
@@ -749,7 +749,7 @@ class Evaluator:
     It allows for the sanitisation of frontend inputs.
     """
 
-    template_regex = re.compile(r"[\\]?@{([\w\s.\[\]]*)}")
+    template_regex = re.compile(r"[\\]?@{([\w\s.\[\]]*)}", re.U)
```

Comment:
As I eventually found out, this flag is redundant and included for backward compatibility.